doc/01_index.md

```diff
@@ -8,8 +8,8 @@ Some general hints:
 
 * No matter what's the current keyboard layout or on which platform the
   game is running, the console is always opened and closed by pressing
-  **Left-Shift + Escape**.
-* Yamagi Quake II saves it's configuration, savegames and screenshots
+  <kbd>Left Shift + Escape</kbd>.
```

Comment:
I don't think using `<kbd>` is really an improvement here (and it looks worse when viewing the file as a plain text file)

Reply:
I reverted to bold instead of using `<kbd>`.